web-test-runner.config.mjs

```diff
@@ -12,6 +12,7 @@ export default {
   coverageConfig: {
     include: ['src/**.js'],
   },
+  testsFinishTimeout: 20000,
```

Comment:
Take a look at this file to see how we should be changing the timeout times:
https://github.com/advanced-rest-client/api-request/blob/main/web-test-runner.config.mjs#L14